core/src/main/java/tc/oc/pgm/structure/Dynamic.java

```diff
@@ -0,0 +1,69 @@
+package tc.oc.pgm.structure;
+
+import org.bukkit.util.Vector;
+import tc.oc.pgm.api.feature.Feature;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.util.BudgetWorldEdit;
+
+public class Dynamic implements Feature<DynamicDefinition> {
+
+  private final Match match;
+  final StructureDefinition structureDefinition;
+  final Vector offset;
```

Comment:
```suggestion
  private final StructureDefinition structureDefinition;
  private final Vector offset;
```

make them accessible via getter if they're needed elsewhere